test/conversion.test.js

```diff
@@ -27,6 +27,18 @@ describe('Curl converter should', function() {
 		});
 	});
 
+	it('[Github: #1]: not throw an error for having $ before method', function (done) {
+		convert({
+			type: 'string',
+			data: `curl -i -s -k  -X $'POST' \
```

Comment:
if you're testing for `$ before method`, use the *simplest* input that tests for that. No headers/body needed.